utils/swift_build_support/swift_build_support/targets.py

```diff
@@ -40,4 +40,8 @@ def host_target():
         if machine == 'amd64':
             return 'freebsd-x86_64'
 
+    elif system == 'CYGWIN_NT-10.0':
```

Comment:
I used Windows 10. Yes, It depend on Windows releases. I found https://en.wikipedia.org/wiki/Uname .